core/modules/replica/ChunkLocker.h

```diff
@@ -119,47 +125,60 @@ class ChunkLocker {
      * Find chunks which are locked by a particular owner (if provided),
      * or by all owners.
      *
-     * @param owner - an optional owner. If the owner is not provided then
-     *                all chunks will be returned
+     * @param owner
+     *   an optional owner. If the owner is not provided then
+     *   all chunks will be returned
      *
-     * @return a collection of chunks grouped by owners
+     * @return
+     *   a collection of chunks grouped by owners
      */
     OwnerToChunks locked(std::string const& owner=std::string()) const;
 
     /**
      * Lock a chunk to a specific owner
      *
-     * @param chunk - a chunk to be locked
-     * @param owner - an identifier of an owner claiming the chunk
+     * @param chunk
+     *   a chunk to be locked
      *
-     * @return 'true' of the operation was successful or if the specified
-     * owner already owns it, or 'false' if there is outstanding lock on
-     * a chunk made earlier by another owner.
+     * @param owner
+     *   an identifier of an owner claiming the chunk
      *
-     * @throw std::invalid_argument - if the owner 'id' is an empty string
+     * @return
+     *   'true' of the operation was successful or if the specified
+     *   owner already owns it, or 'false' if there is outstanding lock on
+     *   a chunk made earlier by another owner.
+     *
+     * @throw std::invalid_argument
+     *   if the owner 'id' is an empty string
      */
     bool lock(Chunk const&       chunk,
               std::string const& owner);
 
     /**
      * Release a chunk regardless of its owner
      *
-     * @param chunk - a chunk to be released
+     * @param chunk
+     *   a chunk to be released
      *
-     * @return 'true' if the operation was successful
+     * @return
+     *   'true' if the operation was successful
      */
     bool release(Chunk const& chunk);
 
     /**
      * Release a chunk and, if successful, set an identifier of an owner which
      * previously 'claimed' the chunk.
      *
-     * @param chunk - chunk to be released
-     * @param owner - reference to a string which will be initialized with
-     *                an identifier of an owner which had a claim on the chunk
-     *                at a time of the method call
+     * @param chunk
+     *   chunk to be released
+     *
+     * @param owner
+     *   reference to a string which will be initialized with
+     *   an identifier of an owner which had a claim on the chunk
+     *   at a time of the method call
```

Comment:
Yes, the string will be modified by the method upon a successful completion of the method. That's why it's not `const`. Since you're asking this question then my description didn't make it clear. I will fix it.